### PR TITLE
[feat ] Full Tagging Support and URL Namespacing

### PR DESCRIPTION
**Pull request by hstrowd**
Adds proper namespaces for routes to distinguish between the '/resource' and '/blog' sections of the site. Updates blog posts to have an array of tags instead of a single tag. Supports posts having multiple tags across the system. Gatsby/graphql required the data type for tags to be consistent across all posts, for this reason I had to update the tags entry in the frontmatter for all posts to use an array even when only one entry was present.

Basecamp To-Dos:
* https://3.basecamp.com/4029961/buckets/7988885/todos/1608868740
* https://3.basecamp.com/4029961/buckets/7988885/todos/1608861010

@hatchloyalty/engineering

### CODE REVIEW

File: src/templates/tag-list.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import Wrapper from "../components/wrapper"
+
+const TagListCard = system({
+  is: Panel,
+  px: 4,
+  py: 4,
+  my: 4,
+})
+
+class TagsListTemplate extends React.Component {
```
Comment (hstrowd) [inline src/templates/tag-list.js]: @DLavin23 I found that I had to pull this component out of `/src/pages` to introduce the `/blog` namespace into the URL. Let me know if I'm overlooking an established Gatsby pattern for managing the routes of pages in another way. The functionality surrounding entries in `/src/pages` is still a bit of a black box to me. Thanks.

File: content/blog/2017-04-24-innovation-in-fuel-c-store-technology-and-consumer-engagement/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "Innovation in Fuel & C-Store Technology and Consumer Engagement"
 date: "2017-04-24"
 author: "Natasha Janic"
 featuredImage: "./Executive-Innovation-Panel.jpg"
-tags: "Enterprise Loyalty"
+tags: ["Enterprise Loyalty"]
```
Comment (DLavin23) [inline content/blog/2017-04-24-innovation-in-fuel-c-store-technology-and-consumer-engagement/index.md]: 👍

File: src/templates/tag-list.js (hunk 1 above)
Comment (DLavin23) [reply src/templates/tag-list.js]: Unfortunately, I haven't gotten too deep into routing with Gatsby yet, so I don't really know the best patterns. I'm sure what you've done here is fine for now.

Comment (DLavin23) [reply src/templates/tag-list.js]: The only other pattern I've seen with regards to routing in Gatsby is specifying the path you want in the frontmatter of each post. Then in the gatsby-node file, you add a prefix + the path to the frontmatter.

Comment (hstrowd) [reply src/templates/tag-list.js]: Thanks for the heads up. I'll keep an eye out for any other established patterns in this area as well.

**Pull request merged**